HIP/hip-0000-nft-standard.md

```diff
@@ -0,0 +1,795 @@
+	"sha256_checksum": "ba7816bf8f01cfea414140de5dae2223b00361a396177a9cb410ff61f20015ad",
+	"type": "image/png",
+	"format": "HIP412",
+	"properties" : {},
+	"files": [
+		// Localized files (Dutch)
+		{
+			"uri": "ipfs://bawlkjaklfjoiaefklankfldanmf-dutch",
+			"sha256_checksum": "ba7816bf8f01cfea414140de5dae2223b00361a396177a9cb410ff61f20015ad",
+			"type": "image/jpeg"
+		},
+		{
+			"uri": "ipfs://slaijfidnsklwnwklwnwlbawlkjb-dutch",
+			"sha256_checksum": "6db3e42df27a8c6c63a2ac8396412d585c749ef84a4b5d9fced5b891ea9fb853",
+			"type": "image/jpeg"
+		},
+		{
+			"uri": "ipfs://nsklwlaijfidbawakssaioalikwa-dutch",
+			"is_default_file": true,
+			"sha256_checksum": "b61127d81eda21b0eb789be531e76c986717b4756620666095fe19082db9d426",
+			"type": "image/jpeg"
+		}
+	],
+	"attributes": [
+		{
+			"trait_type": "kleur",
+			"value": "rood"
+		},
+		{
+			"trait_type": "mond",
+			"value": "bubblegum"
+		},
+		{
+			"trait_type": "coolheid",
+			"value": 50
+		}
+	]
+	// Don't define a localication object here to avoid infinite localization loops pointing to each other
+}
+```
+
+
+## Formal JSON Schema Definition
+
+The following is the formal definition of this schema using JSON Schema notation. JSON Schema assists in metadata validation and describes the data in the schema in more formal terms. **Despite looking like JSON, this is NOT how actual metadata should look. If you are creating the metadata file for an NFT do not use this definition as a reference.** For more info, see [JSON-schema.org](https://json-schema.org/).
+
+```json
+{
+	"$schema": "http://json-schema.org/draft-04/schema#",
+	"type": "object",
+	"properties": {
+		"name": {
+			"type": "string",
+			"description": "Identifies the asset to which this token represents."
+		},
+		"creator": {
+			"type": "string",
+			"description": "Identifies the artist name(s)."
+		},
+		"creatorDID": {
+			"type": "string",
+			"format": "uri",
+			"description": "Points to a decentralized identifier to identify the creator.",
+		},
+		"description": {
+			"type": "string",
+			"description": "Describes the asset to which this token represents."
+		},
+		"image": {
+			"type": "string",
+			"format": "uri",
+			"description": "A URI pointing to a preview image resource."
+		},
+		"sha256_checksum": {
+			"type": "string",
+			"description": "Cryptographic hash of the representation of the 'image' resource."
+		},
+		"type": {
+			"type": "string",
+			"description": "Sets the MIME type for the 'image' resource."
+		},
+		"format": {
+			"type": "string",
+			"description": "Name of the format or schema used by the NFT."
+		},
+		"properties": {
+			"type": "object",
+			"properties": {
+				"external_url": {
+					"type": "string",
+					"format": "uri",
+					"description": "A URI pointing to an informational page about the NFT."
+				}
+			},
+			"additionalProperties": true
+		},
+		"files": {
+			"type": "array",
+			"contains": {
+				"type": "object"
+			},
+			"minContains": 1,
```

Comment:
As mentioned above, I don't think we need to force a files array if the image is enough for the NFT

Reply:
I would opt to force this because otherwise we will get a mix between people who use the preview image as the main image and some who don't. If you look at other specifications, they even provide guidelines for maximum dimensions for the preview image? We wanted to be a bit more specific so I don't think this hurts?